zbus_names/src/bus_name.rs

```diff
@@ -194,7 +194,24 @@ impl<'name> From<WellKnownName<'name>> for BusName<'name> {
     }
 }
 
-/// Try to create an `BusName` from a string.
```

Comment:
why remove this comment? :thinking:

Reply:
For consistency - `TryFrom<String>`, `TryFrom<Arg<str>>`, etc. did not have this comment. And I didn't see much value in this comment. In general `///` comments for trait implementations aren't very useful as it should be clear what the trait does anyway.

Reply:
Right but please do this in a separate commit and add this comment as justification.

Reply:
Nice! Hate to bring it up but you forgot the break the long line in the commit log. Since rest of the commits break at 74, this will look very strange there.